src/DataTypes.ts

```diff
@@ -19,3 +23,19 @@ export interface MarkerFileData {
     frames: Array<MarkerFrame>;
 }
 
+// -------------------- forces ---------------------
+
+export interface Force {
+    position: Point3D;   //px, py, pz
+    components: Point3D; //vx, vy, vz
+    torque: number;
+}
+
+export interface ForceFrame {
+    time: number;
+    forces: Array<Force|null>; //usually 1, at most 2
```

Comment:
When would a force be null as opposed to just not being present?

Reply:
I was following the pattern we used for MarkerFrames, where a missing Point3D could be represented as null. I guess since Forces don't need to be uniquely identified by column/index/label like Markers do, I could remove this.